sktime/transformations/series/boxcox.py

```diff
@@ -211,8 +211,14 @@ def _inverse_transform(self, X, y=None):
 class LogTransformer(BaseTransformer):
     """Natural logarithm transformation.
 
-    The natural log transformation can used to make data more normally
-    distributed and stabilize its variance.
+    Transforms each data point x to (scale * log(x+offset))
+
+    Parameters
+    ----------
+    offset : float , default = 0
+             Adds a constant to all the data points and shifts them along the x-axis
+    scale  : float , default = 1
+             Multiplies a constant to all the data points.
```

Comment:
perhaps better "multiplicative scaling constant applied to all the data", since the scale does not multiply (transitive verb) but is the thing that the data is being multiplied with